lib/hotel.rb

```diff
@@ -0,0 +1,51 @@
+require_relative 'reservation'
+
+class Hotel
+  attr_reader :rooms, :reservations
+  def initialize
+    @rooms = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20]
+    @reservations = []
+  end
+
+  def find_available_room
```

Comment:
Find an available room on what date?  How can you know if a room is available if you don't know **when**.